srcpkgs/badwolf/template

```diff
@@ -1,22 +1,25 @@
 # Template file for 'badwolf'
 pkgname=badwolf
-version=1.2.2
+version=1.3.0
 revision=1
 build_style=configure
+configure_args="PREFIX=/usr WITH_WEBKITGTK=4.1 WITH_URI_PARSER=guri"
+make_cmd=ninja
 make_check_target=test
-hostmakedepends="pkg-config gettext"
-makedepends="webkit2gtk-devel libsoup-devel"
+hostmakedepends="pkg-config gettext ninja ed"
+makedepends="libwebkit2gtk41-devel"
+depends="bubblewrap xdg-dbus-proxy"
```

Comment:
These can be removed as they are dependencies of webkitgtk and not badwolf itself.